src/libraries/System.ComponentModel.Composition.Registration/tests/System/ComponentModel/Composition/Registration/RegistrationBuilderTests.cs

```diff
@@ -45,7 +45,7 @@ public DiscoveredCatalog()
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/32743", TestRuntimes.Mono)]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/32743", TestPlatforms.tvOS)]
```

Comment:
@radical I don't think this is an ActiveIssue.  (or at least 32743 is misleading).  I don't think we can support MakeGenericType for non-RuntimeType types in FullAOT (without interpreter fallback).

I think this should be `[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotMonoAot))]`

Reply:
Yep, this is not tvOS specific. Using `RuntimeFeature.IsDynamicCode` or whatever flag NativeAOT uses would be better

/cc @jkotas

Reply:
I guess that's

https://github.com/dotnet/runtime/blob/3e4c4a4c2c3343ac545bf48b9c3189556bfecf73/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs#L109

```
[ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsReflectionEmitSupported)]
```

Which brings up the question: do we run these tests with NativeAOT? I would expect them to fail there, too